tools/oesgx/oesgx.c

```diff
@@ -128,5 +130,26 @@ int main(int argc, const char* argv[])
         }
         printf("MaxEnclaveSize_64: 2^(%d)\n", (regs.edx >> 8) & 0xFF);
     }
+
+    /* Enumeration of Intel SGX Capabilities: figure out EPC size */
+    {
+        Regs regs = {SGX_CAPABILITY_ENUMERATION, 0, 0x2, 0};
+        uint64_t epcSz = 0;
+
+        result = _CPUID(&regs);
+        if (result)
+        {
+            printf("Read SGX_CAPABILITY_ENUMERATION failed:\n");
+            dump_regs(&regs);
+            return result;
+        }
+        if (!HAVE_EPC(regs)) {
+            printf("No EPC section\n");
+            dump_regs(&regs);
+            return 0;
+        }
+        epcSz = ((regs.ecx & 0x0fffff000) | ( (uint64_t)(regs.edx & 0x0fffff) << 32));
+        printf("EPC size=%lu\n", epcSz);
```

Comment:
How about adding that this is the EPC size on the platform? 
Change **EPC size** to **EPC size on the platform**